sql/hive/src/main/scala/org/apache/spark/sql/hive/client/IsolatedClientLoader.scala

```diff
@@ -179,7 +179,7 @@ private[hive] class IsolatedClientLoader(
     val isHadoopClass =
       name.startsWith("org.apache.hadoop.") && !name.startsWith("org.apache.hadoop.hive.")
 
-    name.contains("slf4j") ||
+    name.startsWith("org.slf4j") ||
     name.contains("log4j") ||
```

Comment:
do we need the same for "log4j"?

Reply:
Yes, It looks like a similar issue. I'll add it.

Reply:
Fixed